.github/workflows/benchmarks.yml

```diff
@@ -80,6 +65,11 @@ jobs:
       - name: Install asv
         run: pip install asv
 
+      - name: Fetch from upstream master
+        run: |
+          git remote add upstream https://github.com/tardis-sn/tardis
+          git fetch upstream master
```

Comment:
this is what the checkout does right? Perhaps you could just do git remote add upstream URL?

Reply:
what do you mean? git remote add upstream url is already present.

Reply:
The first step just does this, perhaps the remote is origin there. If yes then we can perhaps reduce this step here.

Reply:
doesn't work. 

https://github.com/officialasishkumar/tardis/actions/runs/9779319538/job/26998276265?pr=46

Reply:
By default I think it is origin, so it does not find upstream, just rename origin to upstream if you don;t want origin

Reply:
updated

Reply:
Thanks